packages/common/src/core/manager.ts

```diff
@@ -0,0 +1,28 @@
+import { AppConfig, Dependencies } from '../config/config.js';
+import dictionaryRouters from '../routers/dictionaryRouter.js';
+
+import { connect } from '../config/db.js';
+import { getLogger } from '../config/logger.js';
+import getCategoryUtils from '../utils/categoryUtils.js';
+import getDictionaryUtils from '../utils/dictionaryUtils.js';
+
+const manager = (configData: AppConfig) => {
+	const deps: Dependencies = {
+		db: connect(configData.db),
+		logger: getLogger(configData.logger),
+		config: configData,
+	};
+
+	return {
+		getConfig: deps,
+		getRouters: {
+			dictionaryRouters: dictionaryRouters(deps),
+		},
+		getFunctions: {
+			dictionaryFunctions: getDictionaryUtils(deps),
+			categoryFunctions: getCategoryUtils(deps),
+		},
```

Comment:
traditionally, `getWhatever` has been used as a function name, so it seems odd naming these properties.
I propose, once instantiated, these properties can be done like `lyricInstance.routers.dictionary()` etc. ("instance" is just a placeholder word instead of "manager" in this case)

```suggestion
		routers: {
			dictionary: dictionaryRouters(deps),
		},
		utils: {
			dictionary: getDictionaryUtils(deps),
			category: getCategoryUtils(deps),
		},
```

Reply:
will definitely fix naming this properties as suggested. Naming is the most difficult part of coding. thanks!